www-apps/miniflux/miniflux-2.0.36.ebuild

```diff
@@ -0,0 +1,740 @@
+	"google.golang.org/genproto v0.0.0-20200729003335-053ba62fc06f/go.mod"
+	"google.golang.org/genproto v0.0.0-20200804131852-c06518451d9c/go.mod"
+	"google.golang.org/genproto v0.0.0-20200825200019-8632dd797987/go.mod"
+	"google.golang.org/grpc v1.2.1-0.20170921194603-d4b75ebd4f9f/go.mod"
+	"google.golang.org/grpc v1.19.0/go.mod"
+	"google.golang.org/grpc v1.20.1/go.mod"
+	"google.golang.org/grpc v1.21.0/go.mod"
+	"google.golang.org/grpc v1.21.1/go.mod"
+	"google.golang.org/grpc v1.23.0/go.mod"
+	"google.golang.org/grpc v1.25.1/go.mod"
+	"google.golang.org/grpc v1.26.0/go.mod"
+	"google.golang.org/grpc v1.27.0/go.mod"
+	"google.golang.org/grpc v1.27.1/go.mod"
+	"google.golang.org/grpc v1.28.0/go.mod"
+	"google.golang.org/grpc v1.29.1/go.mod"
+	"google.golang.org/grpc v1.30.0/go.mod"
+	"google.golang.org/grpc v1.31.0/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200109180630-ec00e32a8dfd/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200221191635-4d8936d0db64/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200228230310-ab0ca4ff8a60/go.mod"
+	"google.golang.org/protobuf v1.20.1-0.20200309200217-e05f789c0967/go.mod"
+	"google.golang.org/protobuf v1.21.0/go.mod"
+	"google.golang.org/protobuf v1.22.0/go.mod"
+	"google.golang.org/protobuf v1.23.0/go.mod"
+	"google.golang.org/protobuf v1.23.1-0.20200526195155-81db48ad09cc/go.mod"
+	"google.golang.org/protobuf v1.24.0/go.mod"
+	"google.golang.org/protobuf v1.25.0/go.mod"
+	"google.golang.org/protobuf v1.26.0-rc.1/go.mod"
+	"google.golang.org/protobuf v1.26.0"
+	"google.golang.org/protobuf v1.26.0/go.mod"
+	"gopkg.in/alecthomas/kingpin.v2 v2.2.6/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15/go.mod"
+	"gopkg.in/errgo.v2 v2.1.0/go.mod"
+	"gopkg.in/resty.v1 v1.12.0/go.mod"
+	"gopkg.in/square/go-jose.v2 v2.5.0"
+	"gopkg.in/square/go-jose.v2 v2.5.0/go.mod"
+	"gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7/go.mod"
+	"gopkg.in/yaml.v2 v2.2.1/go.mod"
+	"gopkg.in/yaml.v2 v2.2.2/go.mod"
+	"gopkg.in/yaml.v2 v2.2.4/go.mod"
+	"gopkg.in/yaml.v2 v2.2.5/go.mod"
+	"gopkg.in/yaml.v2 v2.3.0/go.mod"
+	"gopkg.in/yaml.v2 v2.4.0/go.mod"
+	"gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c"
+	"gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod"
+	"honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod"
+	"honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod"
+	"honnef.co/go/tools v0.0.1-2019.2.3/go.mod"
+	"honnef.co/go/tools v0.0.1-2020.1.3/go.mod"
+	"honnef.co/go/tools v0.0.1-2020.1.4/go.mod"
+	"mvdan.cc/xurls/v2 v2.4.0"
+	"mvdan.cc/xurls/v2 v2.4.0/go.mod"
+	"rsc.io/binaryregexp v0.2.0/go.mod"
+	"rsc.io/pdf v0.1.1/go.mod"
+	"rsc.io/quote/v3 v3.1.0/go.mod"
+	"rsc.io/sampler v1.3.0/go.mod"
```

Comment:
EGO_SUM is deprecated - we should use vendor tarballs (see eclassdocs for go-module.eclass, also gentoo-dev ML discussion)

Reply:
yep, this is part of concord's recruitment quizzes, he'll switch it to the tarball in a followup and host it on dev.g.o once recruitment is finished

Reply:
I figured there was a decent chance of that, but given we've rejected/put PRs on hold for using EGO_SUM, I figured I'd say it now given it's important to be aware of things like this -- we've asked people before to make a github repo or something to dump them in for now, but if concord's told juippis he knows, then it's obviously fine, just wouldn't want it to look like he didn't

Reply:
Using a deps tarball now! :)

Reply:
❤️